### PR TITLE
Create hirschmann_opedge_8d-2.json

### PR DESCRIPTION
**Pull request by Belden-SV**
### Thank you for contributing to the IoT Plug and Play Models repository

:memo: Please review this checklist before submission

- [ ] :eyes: The models submitted in this PR follow the [DTDL Naming Guidelines](https://github.com/Azure/iot-plugandplay-models-tools/wiki/DTDL-Naming-Guidelines) for contributing to this repository.
- [ ] 🕵️I have validated the models locally using the [dmr-client](https://github.com/Azure/iot-plugandplay-models-tools/tree/dev/clients/dotnet/Microsoft.IoT.ModelsRepository.CommandLine) command line tool.
- [ ] ❗ I understand once the PR is approved, the models are considered immutable: any edits will require to submit new files.
- [ ] ✨ When creating a new top level `dtmi:namespace`, I have completed the [PR Info Template](#pr-info-template) below.

:zap: PR validation steps are described in the tools [Wiki](https://github.com/Azure/iot-plugandplay-models-tools/wiki/Validation-Pipeline#pr-validation-checks).

## PR Info Template

When submitting models to the repository we ask that you provide as much of the following meta information around your models and related devices as possible. This info will be used to improve the _IoT Plug and Play_ ecosystem.

👇: Please replace the markdown comment examples with your own values.

**Belden**
https://www.belden.com/

**Hirschmann OpEdge-8D**
https://www.belden.com/products/industrial-networking-cybersecurity/edge-devices/opedge-8d

Model Submission Goal: IoT Plug and Play certification.

This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

### CODE REVIEW

**Comment by Belden-SV** [comment]
@microsoft-github-policy-service agree comapny=Belden

**Comment by rido-min** [comment]
seems you have a typo in the CLA agreement: 
comapny -> company

**Comment by Belden-SV** [comment]
@microsoft-github-policy-service agree company=Belden

File: dtmi/belden/hirschmann_opedge_8d-2.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+		  "@type": "Property",
+		  "name": "version",
+		  "displayName": "Version",
+		  "description": "Version of BHDO.",
+		  "writable": false,
+		  "schema": "double"
+		},
+		{
+		  "@type": "Telemetry",
+		  "name": "value1",
```
Comment (rido-min) [inline dtmi/belden/hirschmann_opedge_8d-2.json:17]: what is `value1` ? I think a better name would be desirable.

Comment (Belden-SV) [reply dtmi/belden/hirschmann_opedge_8d-2.json:17]: Canceling the pull request. Need to revise the model.

**Pull request closed without merging**